styles/modules/modals/_message.scss

```diff
@@ -26,6 +26,10 @@
     .buttonWrap {
       text-align: center;
     }
+
```

Comment:
This seemed like the place to make h1s a tad smaller to match the new design.

Reply:
Should we move this style up a level so it also gets applied to simpleMessage modals?

Reply:
Good point. I'll give that a go and see if it causes any unforeseen issues.